news/13689-install-win-on-unix

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Fix prefix replacement for windows subdir on unixes.
```

Comment:
I think this should either be categorized as a "bug fix" or if you'd like to keep it as an enhancement, word it like, "Adding ability to install Windows packages on unix platforms for cross-compilation".

You also need to link to the pull request at the end by appending `(#13689)` so we can find this and the related issue later.